src/sass/components/RefileWarning.scss

```diff
@@ -9,7 +9,8 @@
 
 .RefileWarning {
   animation: 0.3s ease-out 0s 1 animateWarningHeader;
-  margin: 1px 0 0 0;
+  margin: -1px 0 0 0;
```

Comment:
Instead of the negative margin here we can set the `hr` height to 0 in `EditsNav.scss`. (I think we can actually set the height to 0 in `'hmda.scss`.)